src/renderer/store/modules/settings.js

```diff
@@ -179,12 +179,12 @@ const state = {
   defaultProfile: MAIN_PROFILE_ID,
   defaultQuality: '720',
   defaultSkipInterval: 5,
+  defaultSubtitles: false,
```

Comment:
`defaultSubtitles` makes me feel like default something (e.g. locales) for subtitles, but never linked to "enabled"
```suggestion
  enableSubtitlesByDefault: false,
```